### PR TITLE
[MU4] Fix compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: src/engraving/libmscore/lyrics.cpp [hunk 1]
```diff
@@ -319,8 +319,8 @@ void Lyrics::layout()
 //                   LOGD("create leading, trailing <%s> -- <%s><%s>", qPrintable(text), qPrintable(leading), qPrintable(trailing));
             const TextBlock& tb = textBlock(0);
 
-            const qreal leadingWidth = tb.xpos(leading.size(), this) - tb.boundingRect().x();
-            const int trailingPos = text.size() - trailing.size();
+            const qreal leadingWidth = tb.xpos(static_cast<int>(leading.size()), this) - tb.boundingRect().x();
+            const int trailingPos = static_cast<int>(text.size() - trailing.size());
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/lyrics.cpp:323]: Looks like it's better to update TextBlock::xpos to use size_t, doesn't it?

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/lyrics.cpp:323]: Probably, but that'd be for another PR?

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/lyrics.cpp]: Or another commit ;-)

File: src/engraving/libmscore/lyrics.cpp [hunk 2]
```diff
@@ -320,7 +320,7 @@ void Lyrics::layout()
             const TextBlock& tb = textBlock(0);
 
             const qreal leadingWidth = tb.xpos(leading.size(), this) - tb.boundingRect().x();
-            const int trailingPos = text.size() - trailing.size();
+            const int trailingPos = static_cast<int>(text.size() - trailing.size());
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/lyrics.cpp:323]: That makes this cast unnecessary (change the variable type to size_t)

File: src/engraving/libmscore/textedit.cpp [hunk 3]
```diff
@@ -589,7 +589,7 @@ bool TextBase::edit(EditData& ed)
         score()->undo(new InsertText(_cursor, s), &ed);
 
         int startPosition = cursor->currentPosition();
-        notifyAboutTextInserted(startPosition, startPosition + s.size(), s);
+        notifyAboutTextInserted(startPosition, startPosition + static_cast<int>(s.size()), s);
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/textedit.cpp:592]: Maybe the same can be done here with `notifyAboutTextInserted`.

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/textedit.cpp:592]: Not today anymore

Comment (cbjeukendrup) [reply src/engraving/libmscore/textedit.cpp:592]: Fair enough, seems like that would involve larger changes...

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/textedit.cpp:592]: That's just one reason, the other is that I have a life too 😉

**Pull request merged**